at_client/lib/src/decryption_service/shared_key_decryption.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:at_client/at_client.dart';
+import 'package:at_client/src/decryption_service/decryption.dart';
+import 'package:at_client/src/response/default_response_parser.dart';
+import 'package:at_commons/at_builders.dart';
+import 'package:at_commons/at_commons.dart';
+import 'package:at_utils/at_logger.dart';
+
+/// Class responsible for decrypting the value of shared key's that are not owned
+/// by currentAtSign
+/// Example:
+/// CurrentAtSign: @bob
+/// lookup:phone@alice.
```

Comment:
please remove the full stop after 'alice' in the line above

Reply:
Removed the full stop.